tests/test_validation.py

```diff
@@ -37,6 +37,8 @@
     StringInput,
     VectorInput,
     Option)
+from natcap.invest import validation
```

Comment:
Moved these to top-level imports (see #2170) rather than having to update a lot of them to point to `validation_messages`.